include/globals.h

```diff
@@ -1171,7 +1171,15 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
 #endif
 
 #ifndef ENABLE_WEBSERVER
-#define ENABLE_WEBSERVER        0   // Chip provides a web server with controls to adjust effects
+    #define ENABLE_WEBSERVER        0   // Chip provides a web server with controls to adjust effects
+
```

Comment:
I suggest we just leave this line alone and decrease the diff size a bit. We're not a big fan of whitespace-only changes, and this one breaks the "same indentation #ifndef/#define/#endif" pattern as well.